libvast/src/format/pcap.cpp

```diff
@@ -60,7 +59,21 @@ inline const auto pcap_packet_type = make_packet_type();
 inline const auto pcap_packet_type_community_id = make_packet_type(
   record_field{"community_id", string_type{}.attributes({{"index", "hash"}})});
 
-} // namespace <anonymous>
+} // namespace
+
+void pcap_close_wrapper::operator()(struct pcap* handle) const noexcept {
+  ::pcap_close(handle);
+}
+
+void pcap_dump_close_wrapper::operator()(
+  struct pcap_dumper* handle) const noexcept {
+  ::pcap_dump_close(handle);
+}
+
+void pcap_stat_delete_wrapper::operator()(
+  struct pcap_stat* handle) const noexcept {
+  delete handle;
```

Comment:
I'm surprised there isn't an exported C function for cleaning up the `pcap_stat` handle. I briefly looked around and didn't see one, which is odd I'd imagine.

Reply:
I verified that this doesn't exist.